packages/new-polymath-ui/src/components/inputs/Select/Select.tsx

```diff
@@ -149,7 +149,13 @@ export const SelectPrimitive = Object.assign(ThemedSelectPrimitive, {
   defaultProps: SelectPrimitiveBase.defaultProps,
 });
 
-const EnhancedSelect = formikProxy(SelectPrimitive);
+const EnhancedSelect: FC<FormikExternalProps> = ({ field, form, ...rest }) => (
+  <FormikProxy<any>
```

Comment:
Is this OK having `any` as the value type?

Reply:
It is fine in this scenario. I don't see other way to type this part tbh